### PR TITLE
Wilderness Overhaul

### PR DESCRIPTION
**Pull request by Woodratt**
- Heavy rework of the wilderness, minor adjustments to mining and outpost z-levels. To get to the wilderness you now have to travel through the mine z-level to do so, follow the green flagged path through the mine.
  - Shuttles can now land at a site near the entrance to the wilderness. Removal of the mine shuttle landing pad to move it to the wilderness.
  - New addition of warning sign, thanks to Schnayy.
  - Bugfixes for Heist shuttle not going back to raider post, light overlays on shuttle areas, and certain shuttles not leaving behind floor tiles when they should.

### CODE REVIEW

**Comment by PrismaticGynoid** [comment]
Could we get a map of what the river system will look like now?

**Comment by Woodratt** [comment]
https://i.imgur.com/lan5DQs.png

**Comment by Neerti** [comment]
The NanoUI maps will need to be re-generated after this gets merged as a note.

File: code/defines/obj.dm [hunk 1]
```diff
@@ -203,6 +203,12 @@ var/global/list/PDA_Manifest = list()
 	anchored = 1
 	unacidable = 1//temporary until I decide whether the borg can be removed. -veyveyr
 
+/obj/structure/showcase/sign
+	name = "WARNING: WILDERNESS"
+	icon = 'icons/obj/stationobjs.dmi'
```
Comment (Belsima) [inline code/defines/obj.dm]: Don't need this line. Already set.

File: html/changelogs/Woodrat - Wilderness.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - maptweak: "Heavy rework of the wilderness, minor adjustments to mining and outpost z-levels. To get to the wilderness you now have to travel through the mine z-level to do so, follow the green flagged path. Through the mine."
+  - tweak: "Shuttles can now land at a site near the enterance to the wilderness. Removal of the mine shuttle landing pad to move to the wilderness."
```
Comment (Belsima) [inline html/changelogs/Woodrat - Wilderness.yml]: entrance, not enterance

**Pull request merged**